src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/AttachCaseCallbackService.java

```diff
@@ -56,18 +58,30 @@ public AttachCaseCallbackService(CcdApi ccdApi) {
      * @return Either a map of fields that should be modified in CCD when processing was successful,
      *         or the list of errors, in case of errors
      */
-    public Either<List<String>, Map<String, Object>> process(CaseDetails exceptionRecord) {
+    public Either<List<String>, Map<String, Object>> process(CaseDetails exceptionRecord, String eventId) {
```

Comment:
This service is called "AttachCaseCallbackService".
If eventId is not `attach_to_case` (or whatever it is) we should stop the whole thing straight away, no need to validate eventId + classification combination....

Reply:
and then, validation can be simplified to calling something like
`canBeAttachedToCase(exceptionRecord)`
instead of
`hasValidCombinationOfEventIdAndClassification(exceptionRecord, eventId)`